lib/kafka_ex/server.ex

```diff
@@ -109,9 +110,13 @@ defmodule KafkaEx.Server do
     {:reply, response, state}
   end
 
+  def handle_call({:offset_fetch, offset_fetch}, _from, state = %State{consumer_group: false}) do
```

Comment:
This should get handled by #71 - it will actually error out, which I think is better.  You don't want someone to find out too late that they're operating under invalid assumptions.